ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/embeddedActivityBlock/embeddedActivityBlock.component.ts

```diff
@@ -112,7 +112,7 @@ export class EmbeddedActivityBlockComponent implements OnInit {
             tap(() => this.componentBusy.emit(true)),
             tap(() => {
                 activityValidStatus = this.activity.validate();
-                this.validStatusChanged.emit(activityValidStatus);
+                this.validStatusChanged.emit({id: this.instance.instanceGuid, value: activityValidStatus});
```

Comment:
Block guids and instance guids are generated separately, meaning there is a possibility that they can have the same ids. I doubt we'll ever encounter a conflict, so I'll be okay going along with this, but we might want to make a note of it.

If the ids aren't used for anything other than mapping the value, then perhaps a simple scheme here is to prepend a unique prefix? E.g. `b-` for blocks and `i-` for instances?

Reply:
Added at e51002dec7c5a6c2e7f356d065ca3a9dbf0fa034

Reply:
Do we need `idPrefix.instance` here so things match up?

Reply:
It is added at the top level in `activitySection` component.

`EmbeddedActivityBlockComponent` (**id**) =>
    `ActivityBlockComponent` (**id**) =>
        `ActivitySectionComponent` (**idPrefix.instance + $event.id**) =>

Reply:
Ah, nice! I see it now.